src/modules/battery.cpp

```diff
@@ -1,9 +1,16 @@
 #include "modules/battery.hpp"
-
+#if defined(__FreeBSD__)
+// clang-format off
+#include <sys/types.h>
+#include <sys/sysctl.h>
+// clang-format on
+#endif
 #include <spdlog/spdlog.h>
 
+#include <iostream>
 waybar::modules::Battery::Battery(const std::string& id, const Json::Value& config)
     : ALabel(config, "battery", id, "{capacity}%", 60) {
+#if defined(__Linux__)
```

Comment:
Should be lowercase:
```suggestion
#if defined(__linux__)
```
Also, it's not complete without the same conditional around
https://github.com/Alexays/Waybar/blob/246e377404b2c189d7e78f4b4038ecee3e602dc4/include/modules/battery.hpp#L9

Reply:
Done, thanks.